frontend/src/admin/components/user-list/user-list.tsx

```diff
@@ -1,17 +1,16 @@
-import React, {useState} from "react";
-import Users from "../../users.json";
+import {useState} from "react";
 import {User} from "../../models/user";
+import DropdownMenu from "../sidebar/dropdown/dropdown-menu";
 import UserListItem from "./user-list-item";
-import IconButton from "./icon-button";
-import "./user-list.scss";
+
 
 export default function UserList() {
     const [selectedUser, setSelectedUser] = useState<User>();
     return (
         <section className="c-user-list">
             <div className="c-user-list__header">
                 <h2>Users</h2>
-                <IconButton/>
+                <DropdownMenu/>
             </div>
             <div className="c-user-list__items">
                 {Users.users.map((user) => (
```

Comment:
You have removed the `Users` import statement, but still tries to iterate over it.
I tried switching to this branch on my local machine and there are multiple errors when building. These should be addressed.

Reply:
✅ This is changed